test/adapters/sequelize.test.js

```diff
@@ -0,0 +1,123 @@
+const Sequelize = require('sequelize');
+const getSchema = require('../../src/adapters/sequelize');
+const { sequelizePostgres, sequelizeMySQLMin, sequelizeMySQLMax } = require('../databases');
```

Comment:
[MISC] Should not we do like on lumber ? describeFor... This will ease the writting of tests

Reply:
I guess we could refactor this in another PR (I kept the current implementation of tests, see `index` file). Also I searched for `describeFor` in lumber and did not found anything. Still, I found a mix of specific files, `describe('using mysql' ...`, some kind of loops, etc. In other word it seems it's inconsistent in Lumber (maybe it's not an actual problem though) and I don't feel like we should think about consistence in this actual PR.